Languages/french.lang

```diff
@@ -1948,7 +1948,7 @@ msgid "Save List"
 msgstr "Exporter la liste"
 
 msgid "Save Path"
-msgstr ""
+msgstr "Sauvegarder le dossier"
```

Comment:
```suggestion
msgstr "Chemin des sauvegardes"
```
ou: de la sauvegarde

(this is not the verb save + path noun complement, but path of the save)